### ISSUE TITLE
cant set allowEmpty to true in remark-lint-fenced-code-flag

### ISSUE THREAD

**Issue by Ir1d** [remarkjs/remark-lint#185]
`b.md`:

    ```
    alpha();
    ```

`.remarkrc`:

```
{
  "plugins": [

    ["remark-lint-code-block-style", "fenced"],
    [
      "remark-lint-fenced-code-flag",
      {"allowEmpty": true}
    ],

    
    ["remark-lint-no-file-name-articles", false],
    "remark-pangu",
    "remark-math",
    "remark-lint-final-newline",
    "remark-lint-no-tabs"
  ]
}
```

b.md and .remarkrc are in the same folder, and I am at this foler:

```
remark b.md
```

results:

```
    alpha();
b.md: no issues found
```

**Comment by wooorm** [comment]
The no issues found part is the expected output right? This should happen now, is it not?

**Comment by Ir1d** [comment]
It said no issues found but the code was actually linted?

I wonder if I could make it remain like this.

    ```
    alpha();
    ```

**Comment by Ir1d** [comment]
Thank you so much!

This managed to work:

```
remark().use(styleGuide).use({
  settings: {
    fences: true
  }
}).process(x, function (err, file) {
  console.error(report(err || file));
  console.log(String(file));
});
```

BTW, is it the same with adding this to my `.remarkrc` ?
```
  "settings": {
    "fences": true
  },
```

**Comment by wooorm** [comment]
Yup! [some more info here](https://github.com/unifiedjs/unified-engine/blob/master/doc/configure.md)!

**Comment by wooorm** [comment]
note though that the JS api, so `remark()`, does not read `.remarkrc` and the like, but the cli does!
